views/js/runner/plugins/content/modalFeedback/modalFeedback.js

```diff
@@ -0,0 +1,185 @@
+
+        /**
+         * Initialize the plugin (called during runner's init)
+         */
+        init: function init() {
+            renderedFeedbacks = [];
+            isDestroyed = false;
+            nextStep = function(){};
+
+            defineMode(!!module.config().inlineModalFeedback);
+        },
+
+        /**
+         * Called during the runner's render phase
+         */
+        render: function render() {
+            var self = this;
+            var testRunner = this.getTestRunner();
+
+            var createMessages = function createMessages(renderingQueue, inline){
+
+                var bInlineMode = inlineMode;
+
+                if (inline === true || inline === false) {
+                    defineMode(inline);
+                }
+
+                if (renderingQueue.length) {
+
+                    _.each(renderingQueue, function (renderingToken) {
+
+                        var feedback = messagePlugin(testRunner, testRunner.getAreaBroker());
+                        feedback.init({
+                            dom: renderingToken.feedback.render({
+                                inline: inlineMode
+                            }),
+                            // for alerts will be used #modalMessages container
+                            $container: inlineMode ? renderingToken.$container : null
+                        });
+                        feedback.render();
+
+                        renderedFeedbacks.push(feedback);
+                    });
+
+                    // auto scroll to the first feedback, only for the "inline" mode
+                    if (inlineMode && renderedFeedbacks) {
+                        autoscroll($('.qti-modalFeedback', testRunner.getAreaBroker().getContentArea()).first(), testRunner.getAreaBroker().getContentArea().parents('.content-wrapper'));
+                    }
+                } else {
+                    nextStep();
+                }
+
+                // restore global feedback mode
+                defineMode(bInlineMode);
+            };
+
+            if (inlineMode) {
+                testRunner
+                    .off('plugin-resume.itemInlineMessage')
+                    .on('plugin-resume.itemInlineMessage', function () {
+                        self.destroy();
+                    });
+            } else {
+                testRunner
+                    .off('plugin-resume.itemAlertMessage')
+                    .on('plugin-resume.itemAlertMessage', function (feedback) {
+                        destroyFeedback(feedback);
+                    });
+            }
+
+            testRunner.on('modalFeedbacks', function(renderingQueue, done, inline) {
+                nextStep = done;
+                createMessages(renderingQueue, inline);
+            });
+        },
+
+        /**
+         * Called during the runner's destroy phase
+         * allow to run that function only once
+         */
+        destroy: function destroy() {
```

Comment:
but in the destroyFeedback will be called modalFeedback.destroy before the previous deletion change plugin status